auk-notebook-example.ipynb

```diff
@@ -81,45 +81,46 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# maximum number of words to show in output.\n",
+    "# Maximum number of words to show in output.\n",
     "# Jupyter will create an output error if the number is too high.\n",
     "TOP_COUNT = 30 \n",
     "\n",
-    "# Domain suffixes to check non-U.S. domains.\n",
-    "# so that (e.g.) www.google.co.uk will become \"google\"\n",
+    "# Domain suffixes to check non-U.S. domains so that (e.g.) www.google.co.uk will become \"google\".\n",
     "STOP_DOMAINS = [\"co\", \"org\", \"net\", \"edu\"] # domain suffixes to remove\n",
     "\n",
-    "# minimum number of characters for a word to be included in a corpus\n",
+    "# Minimum number of characters for a word to be included in a corpus.\n",
     "MINIMUM_WORD_LENGTH = 3 # eliminates \"it\", \"I\", \"be\" etc.\n",
     "\n",
-    "# list of substrings to filter a text line, if desired\n",
+    "# List of substrings to filter a text line, if desired.\n",
     "LINE_FILTER = ['404 Not Found']\n",
     "\n",
-    "# The number of the last line of text to ingest\n",
+    "# How many lines of text to use.\n",
     "RESULTS_LIMIT = 2500\n",
     "\n",
-    "# If you want to start ingesting at a different line, you can increase this.\n",
+    "# If you want to start at a different line, you can increase this.\n",
     "# If RESULTS_START is great than RESULTS_LIMIT you will get no results.\n",
     "RESULTS_START = 0\n",
     "\n",
-    "# If you have a large file but want to sample the file more broadly, you\n",
-    "# can increase this value skip to every Nth line.\n",
+    "# If you have a large file but want to sample the file more broadly.\n",
+    "# You can increase this value skip to every Nth line.\n",
     "RESULTS_STEP = 5\n",
```

Comment:
I wonder if RESULTS_STEP should be changed to `1` for the default example?  This var (plus RESULTS_START & RESULTS_LIMIT) is used basically so that we do not potato browsers.  Right now, the analysis takes every 5th line starting from 0 and going to 2500, for a total of 500 lines. Alternately, we could keep RESULTS_STEP the same, and increase RESULTS_LIMIT to get more results.